front/components/assistant/conversation/AgentMessage.tsx

```diff
@@ -583,6 +556,27 @@ export function AgentMessage({
             )}
           </div>
         )}
+        <>
+          {agentMessage.actions
+            .filter((a) => isVisualizationActionType(a))
+            .map((a, i) => {
+              const streamingViz = streamedVisualizations.find(
+                (sv) => sv.actionId === a.id
+              );
+              assert(isVisualizationActionType(a));
```

Comment:
let's remove this

Reply:
Will be done in a follow up PR where we clean up the types.